modules/pool/lib/apr-data-sources/ib-yield-apr-handlers/types.ts

```diff
@@ -0,0 +1,10 @@
+export type TokenApr = {
+  val: number;
+  address: string;
+  group?: string;
+}
+
+export interface AprHandler {
+  readonly group: string;
+  getAprs(): Promise<{ [key: string]: number }>;
+}
```

Comment:
could these types live in the ib-yield-apr-handlers.ts without creating circular dependencies?